modules/custom/engineering_profile_helper/modules/engineering_magazine/lib/scss/components/magazine_navigation/_magazine-navigation.scss

```diff
@@ -1,134 +1,143 @@
 .block--page-title {
-    display: none;
+  display: none;
 }
 
-.section-bgcolor--title.section-editorial-content{
-    background: initial;
-    background-color: #00ece9;
+.section-bgcolor--title.section-editorial-content {
+  background: initial;
+  background-color: #00ece9;
 
-    height: initial;
-    padding: initial;
-    margin-left: calc(50% - 50vw);
-    margin-right: calc(50% - 50vw);
-    width: 100%;
+  height: initial;
+  padding: initial;
+  margin-left: calc(50% - 50vw);
+  margin-right: calc(50% - 50vw);
+  width: 100vw;
+
+  .magazine-landing-nav {
```

Comment:
It isn't vertically centered. I suspect it should.

Reply:
fixed.